### PR TITLE
reuse the same cron registry

### PR DESCRIPTION
**Pull request by phiSgr**

### CODE REVIEW

**Review by callicles — APPROVED** [review]

File: apps/framework-cli/src/cli/routines.rs [hunk 1]
```diff
@@ -346,7 +346,8 @@ fn start_leadership_lock_task(redis_client: Arc<Mutex<RedisClient>>, project: Ar
         let mut interval = interval(Duration::from_secs(LEADERSHIP_LOCK_RENEWAL_INTERVAL)); // Adjust the interval as needed
         loop {
             interval.tick().await;
-            if let Err(e) = manage_leadership_lock(&redis_client, &project).await {
+            let cron_registry = CronRegistry::new().await.unwrap();
```
Comment (cjus) [inline apps/framework-cli/src/cli/routines.rs]: Doesn't this create a new CronRegistry on every pass of the loop?

Comment (phiSgr) [reply apps/framework-cli/src/cli/routines.rs]: omg

File: apps/framework-cli/src/cli/routines.rs [hunk 2]
```diff
@@ -344,9 +344,12 @@ async fn process_pubsub_message(
 fn start_leadership_lock_task(redis_client: Arc<Mutex<RedisClient>>, project: Arc<Project>) {
     tokio::spawn(async move {
         let mut interval = interval(Duration::from_secs(LEADERSHIP_LOCK_RENEWAL_INTERVAL)); // Adjust the interval as needed
+
+        let cron_registry = CronRegistry::new().await.unwrap();
```
Comment (cjus) [inline apps/framework-cli/src/cli/routines.rs:348]: cool this fixes it.

**Review by cjus — APPROVED** [review]

File: apps/framework-cli/src/cli/routines.rs (hunk 2 above)
Comment (phiSgr) [reply apps/framework-cli/src/cli/routines.rs:348]: it's amusing how after some changes i managed to make the same silly mistake lol

**Pull request merged**